### ISSUE TITLE
route_link() keep root url but lose path

### ISSUE THREAD

**Issue by csliangy** [Appsilon/shiny.router#68]
I deployed the sample application @https://appsilon.com/shiny-router-package/ on an nginx server with the following path:
https://www.rooturl.com/shinyproxy, when I click the "home" or "side", the root url (www.rooturl.com) is maintained _**but the path "shinyproxy" is disappeared**_. This only happens on the _**route_link**_. The "change_page" is still working as expected. 

```
library(shiny)
library(shiny.router)

# This generates menu in user interface with links.
menu <- (
  tags$ul(
    tags$li(a(class = "item", href = route_link("home"), "Home page")),
    tags$li(a(class = "item", href = route_link("side"), "Side page"))
  )
)

# This creates UI for each page.
page <- function(title, content) {
  div(
    menu,
    titlePanel(title),
    p(content),
    actionButton("switch_page", "Click to switch page!")
  )
}

# Both sample pages.
home_page <- page("Home page", uiOutput("current_page"))
side_page <- page("Side page", uiOutput("current_page"))

# Creates router. We provide routing path, a UI as
# well as a server-side callback for each page.
router <- make_router(
  route("home", home_page, NA),
  route("side", side_page, NA)
)

# Create output for our router in main UI of Shiny app.
ui <- shinyUI(fluidPage(
  router_ui()
))

# Plug router into Shiny server.
server <- shinyServer(function(input, output, session) {
  router(input, output, session)
  
  output$current_page <- renderText({
    page <- get_page(session)
    sprintf("Welcome on %s page!", page)
  })
  
  observeEvent(input$switch_page, {
    if (is_page("home")) {
      change_page("side")
    } else if (is_page("side")) {
      change_page("home")
    }
  })
})

# Run server in a standard way.
shinyApp(ui, server)
```

**Comment by dokato** [comment]
Sorry for the delayed response. Indeed this is an important fix to be made. Will take care of it soon.

**Comment by dokato** [comment]
This should be fixed now with #75. Feel free to reopen if needed.
